backend/tests/test_optimal_store.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+import json
+import model
+import app
+import test_data
+
+
+def test_single_store(client):
```

Comment:
These tests are flakey, they only pass if they are ran after test_model. Solution would be to load the newly added items from test_data and call save on them in each test

Reply:
should also add "db" to the parameter list. This will clear up the database after the test is ran

Reply:
Done